scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua

```diff
@@ -34,6 +29,12 @@ end;
 -----------------------------------
 
 function onMobEngaged(mob,target)
+	-- Two hours to forced depop

```

Comment:
Don't do that. If you do that they can reset teh timer simply by deagro'ing it for a splti secodn and reclaiming before it depops.